proxies/nodejs/app.ts

```diff
@@ -21,6 +22,22 @@ const handleClient = async (ctx) => {
   }
 }
 
+const handleUser = async (ctx) => {
```

Comment:
might be overkill now, but it would be nice to have these handlers in their own file to separate endpoint functionality as opposed to dumping it all in one file